index.html

```diff
@@ -43,6 +43,54 @@
     <div class="bottom-resize"></div>
 
     <script src="supporter.js"></script><!-- support wnr running -->
+
+    <!-- Add minutes translation fallback after i18n is initialized in supporter.js -->
+    <script>
+        // Make sure 'minutes' translation key exists, or fallback to appropriate full words
+        if (!i18n.__('minutes')) {
+            // Define a mapping for full words based on language
+            const minutesMapping = {
+                'zh-CN': '分钟',   // Simplified Chinese
```

Comment:
Also redundant here. You shall never hard-code anything but should always refer from locales/*.json using i18n.__("") syntax.